src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -619,6 +621,11 @@ async def handle_list_resources_command(
     await respond(text="Incident Resources", blocks=blocks, response_type="ephemeral")
 
 
+async def ack_command(ack: AsyncAck) -> None:
+    """Handles request acknowledgement for slash commands."""
+    await ack()
```

Comment:
We could send a message back to the user here, if we want.

Reply:
In order to maintain current functionality; we should probably return a message similar to what we do now.... e.g. "Your command is running... " Using the mapped command strings as part of the message:

https://github.com/Netflix/dispatch/blob/master/src/dispatch/plugins/dispatch_slack/plugin.py#L181